### DISCUSSION TITLE
why `cleanup = true` not working in kde neon `pkcon update -y --autoremove`

### DISCUSSION THREAD

**Discussion by candrapersada** [topgrade-rs/topgrade#1004] [Q&A] [answered]
why `pkcon update -y --autoremove` not working for Remove unrequired dependencies with `cleanup = true`

**Comment by GideonBear** [comment, marked as answer]
Assuming that command was actually run (you were running the latest version of Topgrade, and the config was correctly read), this is not an issue with Topgrade
